src/pages/data-feeds/starknet/index.md

```diff
@@ -0,0 +1,166 @@
+   Transaction status is: PENDING
+   Transaction is: ACCEPTED_ON_L2
+
+   Stored values are:
+   round_id = 3.402823669209385e+38
+   answer = 6
+   block_num = 613094
+   observation_timestamp = 1673365796
+   transmission_timestamp = 1673365809
+   Done in 99.10s.
+   ```
+
+If the invoke request is successful, you can see the stored values in the [StarkScan testnet explorer](https://testnet.starkscan.co/). Search for your contract by the address. Under the **Read Contract** tab, run a query on the `get_stored_round()` method. For example, you can see the [contract methods](https://testnet.starkscan.co/contract/0x0297d4d4e0dc667c82a452cf809176c50a3e3707408ce39d0f1a1a881d35a83f#read-contract) from the previous output examples.
+
+You can achieve a similar result by running the `starknet-compile`, `starknet declare`, `starknet deploy`, and `starknet invoke` [CLI commands](https://docs.starknet.io/documentation/tools/CLI/commands/), but the scripts are useful for interacting with StarkNet contracts and accounts programmatically.
+
+## Running the off-chain example
+
+This example reads the proxy contract to get the latest values with no account or contract compiling steps required. The [readLatestRoundOffChain.ts](https://github.com/smartcontractkit/chainlink-starknet/tree/develop/examples/contracts/proxy-consumer/scripts/readLatestRoundOffChain.ts) script uses [StarkNet.js](https://www.starknetjs.com/) to make a call directly to the data feed proxy address. By default, the script reads the [LINK / USD feed](https://testnet.starkscan.co/contract/0x02579940ca3c41e7119283ceb82cd851c906cbb1510908a913d434861fdcb245), but you can change the address to read any of the [available data feeds](https://docs.chain.link/data-feeds/price-feeds/addresses?network=starknet) on the StarkNet testnet.
+
+In the `./chainlink-starknet/examples/contracts/proxy-consumer/` directory, run `yarn readLatestRoundOffChain`.
+
+```text
+yarn readLatestRoundOffChain
+yarn run v1.22.19
+
+round_id = 3.402823669209385e+38
+answer = 6
+block_num = 613151
+observation_timestamp = 1673367749
+transmission_timestamp = 1673367801
+Done in 3.09s.
+```
+
+You can achieve a similar result by running the [`starknet call` CLI command](https://docs.starknet.io/documentation/tools/CLI/commands/#starknet_call) and specifying the proxy address, function, and the ABI file for the data feed proxy contract. For this example, the ABI file is available in the repository, but you can also generate the ABI yourself from the [aggregator_proxy.cairo](https://github.com/smartcontractkit/chainlink-starknet/blob/develop/contracts/src/chainlink/cairo/ocr2/aggregator_proxy.cairo) contract source file.
+
+```shell
+starknet call --address 0x2579940ca3c41e7119283ceb82cd851c906cbb1510908a913d434861fdcb245 --function latest_round_data --abi ./contracts/aggregator_proxy_abi.json
+```
+
+The command prints the result:
+
+```text
+0x100000000000000000000000000017c51 6 613174 1673368490 1673368507
+```
```

Comment:
One last minor thing I missed last time: include a "What's Next" link to the contract address, for consistency with the Proof of Reserve and NFT Floor Pricing Feeds pages.

Reply:
Good suggestion. Added it just now.